sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -1675,6 +1696,11 @@ object GpuOverrides {
         new SparkPlanMeta[ProjectExec](proj, conf, p, r) {
           override def convertToGpu(): GpuExec =
             GpuProjectExec(childExprs.map(_.convertToGpu()), childPlans(0).convertIfNeeded())
+
+          override def areAllSupportedTypes(types: DataType*): Boolean = types.forall {
```

Comment:
I don't think we want this one here.  It implies that a Project can return a binary type or accept one as input, which would then mean that we need to be able to copy a binary type to/from the GPU which we cannot do yet.

Reply:
I think the unit tests for byte cast requires copying the binary type from the GPU because without this the unit tests fail.

Reply:
How does that work, because that code is not written yet.

https://github.com/NVIDIA/spark-rapids/blob/f3bb5067f32bbf5c6e1ed4d6005e1374d551f7ae/sql-plugin/src/main/java/com/nvidia/spark/rapids/RapidsHostColumnVector.java#L160-L163

https://github.com/NVIDIA/spark-rapids/blob/branch-0.3/sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuRowToColumnarExec.scala

Reply:
I added it as part of this PR, that is from a different commit. https://github.com/NVIDIA/spark-rapids/pull/886/files#diff-fc1afa3c0aee67b8a7ddafdf7ce51d32105ca119811774d9728d995f8a46f1a2R162

By removing the cast function's explicit test, the `ProjectExec` can safely be disabled and the `getBinary` function removed. The cast functionality would then be implicitly tested as part of the hash functionality rather than as its own test. Is this the preferred solution?

Reply:
`ProjectExec` `BinaryType` support removed along with cast specific tests.